src/test/resources/org/springframework/integration/splunk/config/xml/SplunkOutboundChannelAdapterParserStreamTests-context.xml

```diff
@@ -12,12 +12,15 @@
 
 	<int:channel id="outputToSplunk"/>
 
+	
 	<int-splunk:outbound-channel-adapter
 			id="splunkOutboundChannelAdapter" auto-startup="false" order="1"
 			channel="outputToSplunk" splunk-server-ref="splunkServer"
 			source-type="spring-integration" source="example5"
 			host="test.host" host-regex="test.host.*">
-		<int-splunk:index-writer index="foo"/>
+		<int-splunk:hec-writer host="localhost" port="8088" token="4DBFC24E-19A2-4D31-9055-2139C818DBDD" https="false" 
```

Comment:
Having this and looking to the `HEC` docs we don't need to require `splunk-server-ref` anymore:

> To use EC the first thing you have to do is enable it*. You can do this from the Splunk UI by going to Settings -> Data Inputs -> HTTP Event Collector. Once the dialog opens, click on “Global Settings” and then “Enabled” in the Edit Global Settings dialog. Notice the default port is 8088. EC runs on its own dedicated port. This means you don’t need to expose 8089, the Splunk REST API port in order to make Event Collector accessible from the outside. This new port is specific only for sending events.